.github/workflows/deploy.yml

```diff
@@ -0,0 +1,58 @@
+name: Deploy
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  bintray-upload:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
```

Comment:
I believe explicit JDK setting is only when the program is targetting a Java version newer than the default used by GitHub Actions. Since Java is preinstalled, and the default version used is at least 8, we can probably remove such steps. As per my experience, the program will still compile and run even when targeting older Java versions for compatibility reasons. In case I'm wrong, the build will fail, and we'll change it back then.

Reply:
I've tried running the workflow locally with https://github.com/nektos/act but it threw an error when not explicitly specifying the JDK version